domain/storageprovisioning/state/filesystem.go

```diff
@@ -573,13 +573,14 @@ func (st *State) GetFilesystemAttachmentParams(
 SELECT &filesystemAttachmentParams.* FROM (
     SELECT    sf.provider_id,
               mci.instance_id AS machine_instance_id,
-              sfa.mount_point,
-              sfa.read_only,
+              cs.location AS mount_point,
```

Comment:
Instead of aliasing to mount_point, can we rename the filesystemAttachmentParams.mount_point to location?